src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -70,8 +70,6 @@ public CommandResult execute() {
         try {
         	System.out.println("THis executed");
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.